src/components/ChatInput/ChatInput.tsx

```diff
@@ -10,7 +10,6 @@ import SendIcon from "../icons/SendIcon/SendIcon";
 import { MessageWithThread, ThreadId } from "../../types/message";
 import { randomThreadId, handleKeyDown } from "../../utils/helpers";
 import InputLoading from "./InputLoading/InputLoading";
-import { BEING_SENT } from "../../utils/constants";
```

Comment:
remove being_sent from constants if not used at all

Reply:
I removed it.